src/main/java/gregtech/common/metatileentities/miner/MetaTileEntityLargeMiner.java

```diff
@@ -0,0 +1,685 @@
+            maxArea = this.minerLogic.getMaximumChunkDiameter();
+            areaText = new TextComponentTranslation("gregtech.machine.miner.display.working_area.chunks", area, area);
+        } else {
+            area = this.minerLogic.getCurrentDiameter();
+            maxArea = this.minerLogic.getMaximumDiameter();
+            areaText = new TextComponentTranslation("gregtech.machine.miner.display.working_area", area, area);
+        }
+
+        if (configMode) {
+            areaText.appendText(" ").appendSibling(incrButton(area, maxArea, MinerUtil.DISPLAY_CLICK_AREA_INCR))
+                    .appendText(" ").appendSibling(decrButton(area, 1, MinerUtil.DISPLAY_CLICK_AREA_DECR))
+                    .appendText(" ").appendSibling(previewAreaButton(this.minerLogic.isPreviewEnabled()));
+        }
+
+        textList.add(areaText);
+
+        if (configMode) {
+            int yLimit = this.minerLogic.getYLimit();
+            ITextComponent value;
+            ITextComponent hoverText;
+            if (yLimit > 0) {
+                value = new TextComponentString(String.format("%,d", yLimit));
+                hoverText = new TextComponentTranslation("gregtech.machine.miner.display.y_limit.value_hover_tooltip",
+                        value.createCopy());
+            } else {
+                value = new TextComponentTranslation("gregtech.machine.miner.display.y_limit.no_value");
+                hoverText = new TextComponentTranslation(
+                        "gregtech.machine.miner.display.y_limit.value_hover_tooltip.no_value");
+            }
+            textList.add(new TextComponentTranslation(
+                    "gregtech.machine.miner.display.y_limit", new TextComponentString("")
+                    .appendSibling(incrButton(yLimit, Integer.MAX_VALUE, MinerUtil.DISPLAY_CLICK_Y_LIMIT_INCR))
+                    .appendText(" ").appendSibling(decrButton(yLimit, 0, MinerUtil.DISPLAY_CLICK_Y_LIMIT_DECR))
+                    .appendText(" ").appendSibling(value))
+                    .setStyle(new Style().setHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, hoverText))));
+
+            textList.add(new TextComponentTranslation("gregtech.machine.miner.display.repeat",
+                    toggleButton(this.minerLogic.isRepeat(), this.minerLogic.isWorking(),
+                            MinerUtil.DISPLAY_CLICK_REPEAT_ENABLE, MinerUtil.DISPLAY_CLICK_REPEAT_DISABLE)));
+
+            textList.add(new TextComponentTranslation("gregtech.machine.miner.display.chunk_mode",
+                    toggleButton(this.minerLogic.isRepeat(), this.minerLogic.isWorking(),
+                            MinerUtil.DISPLAY_CLICK_CHUNK_MODE_ENABLE, MinerUtil.DISPLAY_CLICK_CHUNK_MODE_DISABLE)));
+
+            textList.add(new TextComponentTranslation("gregtech.machine.miner.display.silk_touch",
+                    toggleButton(this.minerLogic.isRepeat(), this.minerLogic.isWorking(),
```

Comment:
Looks like all the GUI buttons are passing in `this.minerLogic.isRepeat()` for whether they are on or off, instead of their unique booleans.